packages/playground/compile-wordpress/Dockerfile

```diff
@@ -22,6 +22,32 @@ RUN wget -O wp.zip $WP_ZIP_URL && \
     unzip -q wp.zip && \
     rm wp.zip
 
+# Run extra steps for the develop branch
```

Comment:
The more these Dockerfiles grow the more I want to migrate to a Makefile down the road. Nothing blocking here, just sharing.